HotTips/Tips/EDI_Refactorings.md

```diff
@@ -0,0 +1 @@
+﻿Press **Ctrl+.** to activate a specific refactoring on the selected symbol when available.
```

Comment:
Probably better that we promote Alt+Enter.
Ctrl+. also works, but we should really be pushing Alt+Enter.